modules/ROOT/pages/dashboard-custom-config.adoc

```diff
@@ -81,27 +83,27 @@ image::dashboard-custom-config-rows.png[Dashboard Row Configurations]
 |===
 2+| Row Configuration Options for Custom Dashboards
 | Title | Title for your row.
-| Show Title | For making the title visible to end users.
-| Up Arrow icon | For moving a row up in the dashboard when there is more than one row in the dashboard.
-| Down Arrow icon | For moving a row down when there is more than one row in the dashboard.
-| Trashcan icon | For deleting the row _and any charts_ it contains from the dashboard.
+| Show Title | Make the title visible to end users.
+| Up Arrow icon | Move a row up in the dashboard when there is more than one row in the dashboard.
```

Comment:
Move the cursor? Move the focus? same question for line 88.

Reply:
Move the row